presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoUninstaller.java

```diff
@@ -83,4 +89,18 @@ private static void uninstallUsingRpm(boolean checkDependencies, boolean forceUn
         }
         LOGGER.debug("Successfully uninstalled Presto");
     }
+
+    public static void uninstallUsingTar()
+            throws PrestoManagerException
+    {
+        // TODO: Stop Presto before uninstall
+        try {
+            deleteRecursively(TAR_INSTALLATION_DIR);
```

Comment:
On another note, these two methods seems to be doing the same thing. You could just use one.